main/utils/errors.js

```diff
@@ -1,34 +1,172 @@
 'use strict';
 
-const {dialog, clipboard} = require('electron');
+const path = require('path');
+const os = require('os');
+const {clipboard, shell, app} = require('electron');
 const ensureError = require('ensure-error');
 const cleanStack = require('clean-stack');
+const isOnline = require('is-online');
+const {openNewGitHubIssue} = require('electron-util');
+const got = require('got');
+const delay = require('delay');
 
-const showError = async (error, {title, reportToSentry} = {}) => {
+const {showDialog} = require('../dialog');
+
+const MAX_RETRIES = 10;
+
+const getSentryIssue = async (eventId, tries = 0) => {
+  if (tries > MAX_RETRIES) {
+    return;
+  }
+
+  try {
+    // This endpoint will poll the Sentry API with the event ID until it gets an issue ID (~8 seconds).
+    // It will then filter through GitHub issues to try and find an issue matching that issue ID.
+    // It will return the issue information if it finds it or a partial template to use to create one if not.
+    // https://github.com/wulkano/kap-sentry-tracker
+    const {body} = await got.get(`https://kap-sentry-tracker.vercel.app/api/event/${eventId}`, {json: true});
+
+    if (body.pending) {
+      await delay(2000);
+      return getSentryIssue(eventId, tries + 1);
+    }
+
+    return body;
+  } catch (error) {
+    showError(error);
+  }
+};
+
+const getPrettyStack = (error, isPluginError) => {
+  const pluginsPath = path.join(app.getPath('userData'), 'plugins', 'node_modules');
+  return cleanStack(error.stack, {pretty: true, basePath: isPluginError && pluginsPath});
+};
+
+const getIssueBody = (title, errorStack, sentryTemplate) => `
+${sentryTemplate}<!--
+Thank you for helping us test Kap. Your feedback helps us make Kap better for everyone!
+-->
+
+**macOS version:**    ${os.release()} (darwin)
```

Comment:
Super minor, but we could use https://github.com/sindresorhus/macos-release to use a more user-friendly version here. Not Darwin.